modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc

```diff
@@ -0,0 +1,79 @@
+= Converters and serialization 
+:description: Connectors Converters and serialization
+:page-cloud: true
+
+== Key and value converters
+
+Each Redpanda message is a key and value record. Converters handle serialization and deserialization of data between Redpanda topic and a connector. Record key and value converters are configured separately. It can be specified using `Redpanda message key format` and `Redpanda message value format` properties. Key and value converters can be different.
+
+[NOTE]
+====
+If an external system requires structured data (like BigQuery, Snowflake, SQL database), you need to provide data with schema. Then you need to use Avro Converter, Protobuf Converter or JSON Converter with schema.
+====
+
+== ByteArray Converter
+
+The most primitive and high-throughput converter. Schema is ignored.
+It is a default converter type in Managed Connectors. 
+
+To use the converter, select `ByteArray` option as a key or value message format.
+
+== String Converter
+
+It is a high-throughput converter. Schema is ignored. All data are converted to a String.
+
+To use the converter, select `String` option as a key or value message format.
+
+== JSON Converter
+
+JSON Converter can be specified by selecting `JSON` option as a key or value message format.
+The connector needs a message schema to check message format. Usually, the schema is stored in Schema Registry. 
+
+However, if the Schema Registry cannot be used, embedded JSON schema is an option. Then each message contains a schema. It will result in a large size of a message. Example JSON message with embedded schema:
+[source,json]
+----
+{
+  "schema": {
+    "type": "struct",
+    "fields": [
+      {
+        "type": "int64",
+        "optional": false,
+        "field": "person_id"
+      },
+      {
+        "type": "string",
+        "optional": false,
+        "field": "name"
+      }
+    ]
+  },
+  "payload": {
+    "person_id": 1,
+    "name": "Redpanda"
+  }
+}
+----
+
+If you consume JSON data with no message schema, schema check for the connector can be disabled using `Message key JSON contains schema` or `Message value JSON contains schema` option.
+
+== Avro Converter
+
+Avro converter requires schema in Schema Registry.
+
+See also: xref:manage:schema-reg/schema-reg-overview.adoc[Redpanda Schema Registry].
+
+== CloudEvents Converter
+
+CloudEvents Converter is specific for Debezium PostgreSQL and MySQL source connectors. 
+
+See also: https://debezium.io/documentation/reference/2.2/integrations/cloudevents.html[CloudEvents Converter official documentation]
+
+== Protobuf Converter {badge-cloud-beta}
+
+Protobuf Converter requires schema in Schema Registry.
+Currently the converter supports primitive types in sink connectors only.
```

Comment:
Do we support all primitive types, and if not could we list limitations?

Reply:
Good point, I will make it more specific shortly